### PR TITLE
Rerender 4.5.3

### PR DESCRIPTION
**Pull request by xhochy**
As `easyocr` has a `<=4.5.4` pin on OpenCV, I need to rebuild this rather old version. One of the important fixes is that we need to fix the Python 3.10 build strings.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Opencv hasn't been the hardest to migrate. can you reference the issue that stops compilation? Maybe we can help upstream update their version instead?

**Comment by xhochy** [comment]
It seems the upstream issue has been fixed: https://github.com/JaidedAI/EasyOCR/issues/633

**Comment by xhochy** [comment]
I will give this a tiny bit more love while a colleague of mine will look into getting rid of the upstream pin.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -271,3 +270,4 @@ extra:
     - hajapy
     - ocefpaf
     - hmaarrfk
+    - xhochy
```
Comment (hmaarrfk) [inline recipe/meta.yaml:273]: i guess you might want to add yourself to the main branch too.

Comment (xhochy) [reply recipe/meta.yaml:273]: https://github.com/conda-forge/opencv-feedstock/pull/349

**Comment by xhochy** [comment]
@hmaarrfk This has passed and Cubans weren't that complicated. OK to merge?

**Comment by hmaarrfk** [comment]
> Cubans
???

I'm going to consider you in charge of this branch. You can merge at will.

**Comment by hmaarrfk** [comment]
I would prefer if we didn't do automatic "migrations" on this branch.

**Comment by xhochy** [comment]
> I would prefer if we didn't do automatic "migrations" on this branch.

No, I would try to keep this down to the most minimal work possible and hope that it will only have a very short life.

**Pull request merged**